### DISCUSSION TITLE
Best structure for Dreamcast (multi-file) ROMs

### DISCUSSION THREAD

**Discussion by therobbiedavis** [rommapp/romm#141] [Q&A] [answered]
Unlike the other game ROMs that consist of a single file with a specific extension (i.e. name.gb) Dreamcast ROMs can consist of two different formats, GDI and CDI. The former of which usually also comes with .bin and .raw files. As such it needs to be within its own subfolder for a logical file structure (see image below. 

Since RomM does not search subdirectories off the `system/ROMs` root, is the best practice to archive these files in order to then place them on the root?

![image](https://user-images.githubusercontent.com/7907265/230734780-67ad83c2-8556-43dc-88a8-d4e155bb9024.jpeg)

**Comment by crispybegs** [comment]
not the answer you're looking for, but fwiw i swapped out all my dreamcast roms for single file `.chd`

**Comment by zurdi15** [comment]
Hello guys, right now I am still thinking in the best way possible to manage the multi-file games. As @crispybegs sais, right now RomM doesn't support it and you can convert them to ``.chd`` but the multi-file games support for every platform will be available soon.

You could also compress them to let them appear in RomM as a single-file game as a current alternative

**Comment by crispybegs** [comment]
@therobbiedavis there's a pretty comprehensive single collection of dreamcast .chds on archive.org that works just fine

**Comment by zurdi15** [comment, marked as answer]
Just released [v1.6.3](https://github.com/zurdi15/romm/releases/tag/v1.6.3) with multifile games support!
